src/docker-in-docker/install.sh

```diff
@@ -18,8 +18,8 @@ USERNAME="${USERNAME:-"${_REMOTE_USER:-"automatic"}"}"
 INSTALL_DOCKER_BUILDX="${INSTALLDOCKERBUILDX:-"true"}"
 INSTALL_DOCKER_COMPOSE_SWITCH="${INSTALLDOCKERCOMPOSESWITCH:-"true"}"
 MICROSOFT_GPG_KEYS_URI="https://packages.microsoft.com/keys/microsoft.asc"
-DOCKER_MOBY_ARCHIVE_VERSION_CODENAMES="bookworm buster bullseye bionic focal jammy"
-DOCKER_LICENSED_ARCHIVE_VERSION_CODENAMES="bookworm buster bullseye bionic focal hirsute impish jammy"
+DOCKER_MOBY_ARCHIVE_VERSION_CODENAMES="bookworm buster bullseye bionic focal jammy noble"
+DOCKER_LICENSED_ARCHIVE_VERSION_CODENAMES="bookworm buster bullseye bionic focal hirsute impish jammy noble"
```

Comment:
Can you help write a similar test 👇 for the `noble` image? Thanks! 
https://github.com/devcontainers/features/blob/18c182385da24fde37768ea187612aa5e5063d57/test/docker-in-docker/scenarios.json#L2-L10

Reply:
Better idea, I think we should update the 👇 workflow and add the `mcr.microsoft.com/devcontainers/base:noble` image to the `baseImage` 

https://github.com/devcontainers/features/blob/18c182385da24fde37768ea187612aa5e5063d57/.github/workflows/test-all.yaml#L43-L51